README.md

```diff
@@ -43,6 +43,12 @@ sections = api web
 [api]
 api_url = http://api.openweathermap.org/data/2.5/
 
+[api_validation]
+validate_status_code = True
+validate_headers = True
+validate_body = True
+fail_if_field_is_missing = True
```

Comment:
probably rename to: validate_is_field_missing (to have the same naming for all checks)

Reply:
Fixed